### ISSUE TITLE
[v5] attribute bookmark=true for bookmarked tabs

### ISSUE THREAD

**Issue by jathek** [mbnuqw/sidebery#927]
An attribute `bookmark=true` or `bookmark=false` on `.Tab` so that I can style tabs based on whether they're bookmarked or not. Not sure if this is easily possible. I know Firefox can style the bookmark star in the urlbar based on bookmark state of the page; not sure whether that's available to Sidebery.

**Comment by mbnuqw** [comment]
Not easily (there is no API for this, we need to load all bookmarks, go through them and then mark the tabs), but possible.

**Comment by jathek** [comment]
> Not easily (there is no API for this, we need to load all bookmarks, go through them and then mark the tabs), but possible.

Yeah, I figured the bookmarks would have to be loaded, but I know when they *are* loaded, the bookmarks panel shows a dot for bookmarks that are open in tabs, so I was hoping the reverse would be possible.
